Task08.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+
+using std::cout;
+using std::cin;
+using std::endl;
+
+struct Student {
+    char name[30];
+    int fac_num;
+    float grade;
+};
+
+struct Group {
+    int number_of_students;
+    Student *students = new Student[number_of_students];
+    double avarage_grade;
+    ~Group() {
+        delete[] students;
+    }
+};
+
+Group create_group() {
+    int num;
+    double avarage_grade = 0;
+    cout << "Number of students: ";
+    cin >> num;
+    Group res{num};
+    for (int i = 0; i < num; ++i) {
+        cout << "Name of student " << i + 1 << ": ";
+        cin >> res.students[i].name;
```

Comment:
This way you cannot enter first name and last name;
Use getline
When you enter from the console char* and then you want to enter number you need to use cin.ignore();